src/main/java/first/repository/StudentRepository.java

```diff
@@ -43,30 +41,26 @@ public void insert(Student student) {
             preparedStatement.execute();
             System.out.println( "студент точно в базе" );
         } catch (SQLException e) {
-            e.printStackTrace();
+            System.err.println(e);
         }
     }
 
     public List<Student> show() {
         LinkedList<Student> list = new LinkedList<Student>();
```

Comment:
list is not good name for this variable, could you rename it to students for example?